create-kafka-queues/Dockerfile

```diff
@@ -12,17 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM openjdk:8-alpine
-
-RUN apk update
-#Search for a particular version of dependancies
-#RUN apk search -v 'vim*'
-#RUN apk search -v 'curl*'
-#RUN apk search -v 'bash*'
-
-RUN apk --no-cache add vim=8.1.1365-r0
-RUN apk --no-cache add curl=7.64.0-r4
-RUN apk --no-cache add bash=4.4.19-r1
+FROM openjdk:11@sha256:736dd07fc5cb53060a49d38c86b9e673974510c5a18afc40b1c936475f59aee7
```

Comment:
Does openjdk11 include curl, bash and vim?

Reply:
The openjdk 11 uses the full Debian distribution, so looks to include everything we need already. Whereas the alpine variant is approx 1/8th the size but missing a few bits.

We may re-visit Alpine later to see if we can reduce image size a little, but for now we are going to use a standard base image for all containers.

Reply:
The lifecycle of this container is in the order of seconds, and only on a fresh install.
For all others i could see why we'd standardize to a fat feature-full image, but it seems rather extreme for this job